### PR TITLE
docs(readme): stress that configs are community-maintained

### PR DESCRIPTION
**Pull request by clason**
Also suggest helping test PRs with `needs-testing` label

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 A [collection of common configurations](doc/server_configurations.md) for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
 
-This plugin allows for declaratively configuring, launching, and initializing language servers you have installed on your system. Language server configurations are community-maintained.
+This plugin allows for declaratively configuring, launching, and initializing language servers you have installed on your system. 
+** Disclaimer: Language server configurations are on a best-effort basis and community-maintained. See (#contributions).**
```
Comment (mjlbach) [inline README.md:6]: On a best basis? Feels like missing word

File: README.md [hunk 2]
```diff
@@ -208,3 +209,5 @@ a new configuration for it would be appreciated. You can follow these steps:
       [texlab.lua](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua).
 
 4. Ask questions on our [Discourse](https://neovim.discourse.group/c/7-category/7) or in the [Neovim Gitter](https://gitter.im/neovim/neovim).
+
+You can also help out by testing [PRs with the `needs-testing`](https://github.com/neovim/nvim-lspconfig/issues?q=is%3Apr+is%3Aopen+label%3Aneeds-testing) label) that affect language servers you use regularly.
```
Comment (mjlbach) [inline README.md:213]: Not sure how worth the labels are given the small number of PRs

File: README.md (hunk 1 above)
Comment (clason) [reply README.md:6]: oops, yes

Comment (clason) [reply README.md:6]: "best-effort basis" -- what exactly is missing?

File: README.md (hunk 2 above)
Comment (clason) [reply README.md:213]: True, but it has a signal effect as well.

File: README.md (hunk 1 above)
Comment (mjlbach) [reply README.md:6]: A past tense verb :)

Comment (clason) [reply README.md:6]: OK, I thought about "provided" as well ;) 

(I opted for conciseness instead.)

**Pull request merged**